tests/correctness/float/float-const.expect

```diff
@@ -0,0 +1,22 @@
+{
+  "mem_read": {
+    "data": [
+        4.2
+    ],
+    "format": {
+      "is_signed": true,
+      "numeric_type": "floating_point",
+      "width": 32
+    }
+  },
+  "mem_write": {
+    "data": [
+        4.2
```

Comment:
Oh the second thought is that it only proves that it reads/writes the same value, but doesn't prove that it's not necessarily intepretting `4.2` as the decimal representation..

Reply:
Can you explain what you mean by that? The comment is lacking enough context to make sense to me.